phabfive/diffusion.py

```diff
@@ -295,13 +295,15 @@ def edit_uri(
     # TODO: add support for repo_shortname, etc, see get_branches()
     # TODO: add support for handling active vs inactive repos
     # TODO: add support for handling hidden URIs
-    def get_uris(self, repo_id=None):
+    def get_uris(self, repo_id=None, clone=None):
```

Comment:
Rename `clone` to `clone_uri` to make it even more clear that we do not mean clone as in "an action".